tools/appsre-build-fedora-worker-packer.sh

```diff
@@ -8,4 +8,20 @@ export BUILD_RPMS=false
 export SKIP_TAGS="rpmcopy,subscribe"
 export PACKER_ONLY_EXCEPT=--only=amazon-ebs.fedora-38-x86_64,amazon-ebs.fedora-38-aarch64
```

Comment:
1. Why is this `fedora-38`?
2. Can we put the `fedora-38` in a variable and use it here and down below?